src/matrixHelpers/FormioLoader/FormioLoader.test.js

```diff
@@ -0,0 +1,41 @@
+import "@testing-library/jest-dom";
+import { findByText } from "@testing-library/dom";
+import * as FormioLoader from "./index";
+import { testWait } from "../../utils";
+import { formioRes } from "./fixtures";
+
+// Smoke test
+test("FormioLoader is initiated", async () => {
+  const div = document.createElement("div");
+  div.innerHTML = `
+  <div id="formio" 
+    data-formio 
+    data-formio-project-name="dev-svcwlpuksmwawwk" 
+    data-formio-form-name="plsPlusFormDemo" 
+    data-formio-form-confirmation="" 
+    data-formio-form-revision="" 
+    data-formio-env-url="api.forms.platforms.qld.gov.au" 
+    data-formio-pdf-download="no" 
+    data-formio-namespace="" 
+  ></div>
+`;
+  document.body.append(div);
+  FormioLoader.initFormio();
+  jest.spyOn(Formio, "makeRequest").mockResolvedValueOnce(formioRes);
```

Comment:
please add comment to explicitly state that you are stubbing out the curl call to 'env-url' with json payload  inside fixtures folder. As non-javascript ui developers may not know how 'easy' it is to replace functions in javascript under test.